example/src/main/scala/example/Example.scala

```diff
@@ -0,0 +1,84 @@
+package example
+
+import cats.Monad
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import slog.{LoggerFactory, LoggingContext}
+
+class Example[F[_]](
+    loggerFactory: LoggerFactory[F],
+    loggingContext: LoggingContext[F]
+)(implicit F: Monad[F]) {
```

Comment:
can we please have a type which is a subtype of `Logger` and `LoggingContext`, that will be used here in the example?

Reply:
I thought we agreed not to include it in the first version.

Reply:
I think so too. That can always be added once we have the base APIs ready and agreed upon :)